### ISSUE TITLE
Upgrading pretty-format bumped the minimum Node version to 10.14.2

### ISSUE THREAD

**Issue by exogen** [testing-library/dom-testing-library#563]
👋 Hello! I noticed that in between v7.2.2 and v7.3.0, there was a major upgrade to `pretty-format`. Unfortunately, this changes the minimum Node version, so it's possible to minor bump `@testing-library/dom` and not work with the same Node anymore.

- `@testing-library/dom` version: v7.4.0
- Testing Framework and version: jest@24.9.0
- DOM Environment: jsdom@11.12.0

### What you did:

```console
$ yarn
```

### What happened:

```console
$ yarn
error pretty-format@26.0.1: The engine "node" is incompatible with this module. Expected version ">= 10.14.2". Got "10.13.0"
error Found incompatible module.
```
### Problem description:

Due to the `pretty-format` major version bump in #546, the minimum Node version changed to 10.14.2 in between `@testing-library/dom` v7.2.2 and v7.3.0.

Given the value in `engines` and relatively recent #480, it seems intentional to support older Node 10 versions. (For reference, I ran into this because my current client happens to use images containing v10.13.0 for our deploys.)

### Suggested solution:

Downgrade `pretty-format` back to v25, or update this package's stated support.

**Comment by kentcdodds** [comment]
I think we'll probably need to downgrade as suggested and then we can upgrade in the next major version bump.

A PR is welcome.

**Comment by exogen** [comment]
Oops, forgot to mark Fixed in #571. 🎉
